reference/sectionC_interpretation.rst

```diff
@@ -23,18 +29,20 @@ Interpretation of imports
 Units reference
 ---------------
 
-#. A units reference SHALL be a CellML identifier and SHALL be interpreted dependent on the context of the :ref:`CellML model<specA_cellml_model>` in which it occurs, according to the units referencing rules defined later in this section.
-
-#. A :ref:`CellML infoset<specA_cellml_infoset>` MUST NOT contain a units reference for which no referencing rule can be held to have been followed.
+#. A units reference SHALL be a CellML identifier and SHALL be  interpreted dependent on the context of the :ref:`CellML model<specA_cellml_model>` in which it occurs.
 
 #. The units referencing rules are:
 
-   #. Where there is a :code:`units` element with a :code:`name` attribute identical to the units reference, then the units reference SHALL refer to that :code:`units` element.
+   #. Where there is a :code:`units` element with a :code:`name` attribute identical to the units reference within the same infoset, then the units reference SHALL refer to that :code:`units` element.  
```

Comment:
Removes spaces at end of line

Reply:
I don't like the "within the same infoset" bit! It's true, but it make the sentence "Where there is THING with FILTER in X", but you could also read it as "Where there is THING with (FILTER IN X)"

Reply:
Agreed.  Have changed it to: Where, within the same infoset, there is a :code:`units` element with a :code:`name` attribute identical to the units reference, then the units reference SHALL refer to that :code:`units` element.

Reply:
Thanks!